senior-knights/course-schedulizer
# Check to make sure the meetings are not the same section for conflicts

There was a bug with the conflicts where 2 meetings for the same section would appear as a conflict. This PR adds a check to make sure the two meetings are not for the same section before marking it as a conflict. It also adds a utility function to get a string of the location (building and room number separated by a space).
To test:
- Go to develop and load this csv: https://raw.githubusercontent.com/kvlinden/kvlinden.github.io/master/data/full_schedule_2021-04-07_submitted.csv
- Notice that there are sections with red borders that aren't conflicts, such as DATA-202 (they're marked as a conflict because they have 2 rooms)
- Now, load the same csv on this branch and the faulty conflicts should be gone
